pvlib/iotools/psm3.py

```diff
@@ -296,12 +330,25 @@ def parse_psm3(fbuf):
     tz = 'Etc/GMT%+d' % -metadata['Time Zone']
     data.index = pd.DatetimeIndex(dtidx).tz_localize(tz)
 
+    if map_variables is None:
```

Comment:
We should add a test that these warnings are emitted correctly

Reply:
Done!